packages/storybook/stories/Transfer.stories.mdx

```diff
@@ -0,0 +1,76 @@
+import { Meta, Source } from '@storybook/addon-docs';
+import { DetailedBetaBanner } from './BetaBanners/DetailedBetaBanner';
+
+import { TransferSnippet } from './snippets/transfer/Transfer.snippet';
+import { TransferWithDialogPromptSnippet } from './snippets/transfer/TransferWithDialogPrompt.snippet';
+import TransferSnippetText from '!!raw-loader!./snippets/transfer/Transfer.snippet.tsx';
+import TransferWithDialogPromptSnippetText from '!!raw-loader!./snippets/transfer/TransferWithDialogPrompt.snippet.tsx';
+
+<Meta
+  id="transfer"
+  title="Concepts/Transfer"
+  parameters={{ previewTabs: { canvas: { disable: true, hidden: true } } }}
+/>
+
+# Transfer
+
+<DetailedBetaBanner version={'1.5.2-beta.1'} />
+
+During an active call, participants can transfer another participant to another user or number. The transfer process
+starts when one participant requests a transfer to the another participant which they can accept or reject. In the
+current beta version, ACS users cannot request to transfer other participants yet but they can receive transfer
+requests. Right now only Teams users can request to transfer other participants.
+
+A call with a Teams user can be made by using the `CallParticipantsLocator` as the locator
+in the [AzureCommunicationCallAdapterArgs](./?path=/docs/composite-adapters--page#azurecommunicationcalladapterargs)
+when using the [useAzureCommunicationCallAdapter](./?path=/docs/composite-adapters--page#useazurecommunicationcalladapter)
+function to create your [CallAdapter](path=/docs/composite-adapters--page#call-adapter).
+In the current beta version, only one Teams user can be called.
+
+The id of a Teams user can be obtained by using the
+[Microsoft Graph Explorer](https://developer.microsoft.com/en-us/graph/graph-explorer).
```

Comment:
Is there not documentation that we can link out to here?

Reply:
I also thought you were going more in depth here so happy to resolve this. linking out to this I think is correct